juliaup-init.sh.cmd

```diff
@@ -0,0 +1,946 @@
+@{' 2>nul&::'={\"<<__sh__ 2>/dev/null;#"}}[0]<#
+
+
+
+:: ******************************************************************
+:: (1) juliaup-init bat version (Powershell trampoline)
+:: ******************************************************************
+@echo off
+setlocal 
+set "_arg1=%~1" &set "_arg2=%~2" &set "_arg3=%~3" &set "_arg4=%~4" &set "_arg5=%~5"
+powershell -NoLogo -NoP -Exec ByPass "iex (Get-Content '%~f0' -Raw)"
+
+:: Pause if this is a cmd pop-up from Explorer
+if /i "%comspec% /c __%~0_ _" equ "%cmdcmdline:"=_%" pause
+goto :EOF
+
+
+
+__sh__
```

Comment:
So... this way of combining `sh` and `cmd` and `powershell` within one script is clever and entertaining. IIUC

* In `sh` mode, this first part is treated as a heredoc and skipped.
* As a cmd script, the part above is a cmd bootstrapper to run powershell with the content of the file
* As a powershell script, the whole first part is skipped and the script starts at line 700 

It's pretty amazing that can be made to work, but practically it also seems unnecessary and complicated!

Wouldn't it be cleaner just to have one script file for windows (cmd+powershell) and another for unix? I guess I just don't see the advantage in combining them. This would also reduce divergence with the original unix script at https://github.com/rust-lang/rustup/blob/master/rustup-init.sh

Reply:
Will split to `julia-init.sh` and `julia-init.cmd`